src/custom/pages/Profile/index.tsx

```diff
@@ -28,9 +28,12 @@ import NotificationBanner from 'components/NotificationBanner'
 import { SupportedChainId as ChainId } from 'constants/chains'
 import AffiliateStatusCheck from 'components/AffiliateStatusCheck'
 import { useHasOrders } from 'api/gnosisProtocol/hooks'
+import { shortenAddress } from '@src/utils'
```

Comment:
Don't use `@src` imports.

Probably will work without it (I have not tested it)

```suggestion
import { shortenAddress } from 'utils'
```